Checks/PWD007/README.md

```diff
@@ -53,6 +55,51 @@ void foo(int *x, int *y, int size) {
 }
 ```
 
+#### Fortran
+
+The following code performs an exclusive scan, naively parallelized using
+multithreading:
+
+```f90
+subroutine foo(x, y)
+  implicit none
+  integer, intent(in) :: x(:)
+  integer, intent(inout) :: y(:)
+  integer :: i
+
+  y(1) = 0
+
+  !$omp parallel do
+  do i = 2, size(y, 1)
+    y(i) = y(i - 1) + x(i - 1)
+  end do
+  !$omp end parallel do
+end subroutine foo
+```
+
+This is incorrect due to the recurrence pattern present in the code, in the
+form dependencies between two consecutive loop iterations. For instance, it can
+be safely implemented in OpenMP 5.0 using the `scan` directive:
```

Comment:
The first sentence is not correct. Revisit the wording please!

Reply:
[Done](https://github.com/codee-com/open-catalog/compare/890090fa657d060c483661162e52a5728f727572..3b747d1fb40a3e4c1b0a8c7ea65576619d35163c)! Updated the C code example wording as well, as they are a copy of each other.

Reply:
TYSM 🫡